helpers/python/requirements.txt

```diff
@@ -1,6 +1,6 @@
 pip==18.1
 pip-tools==3.1.0
 hashin==0.13.4
-pipenv==2018.7.1
+pipenv==2018.10.9
```

Comment:
> Should fix many Python bugs

Suddenly, it introduces more bugs than you can expect :smile:

Reply:
Haha, yeah, that's definitely a danger...! 😉